server/src/main/java/org/eclipse/lsp/cobol/core/visitor/VariableUsageDelegate.java

```diff
@@ -38,6 +39,16 @@
 @Slf4j
 @RequiredArgsConstructor
 public class VariableUsageDelegate {
+
+  /** IMPLICIT FIELDS, resolved by DB2 pre-compiler or co-processor */
+  public static final Set<String> IMPLICIT_FIELDS = new HashSet<>();
```

Comment:
You may use EnumSet here.

Reply:
> You may use EnumSet here.

EnumSet works only with ENUM-CONST not with values. I need values consisting dashes